torch/onnx/symbolic_opset9.py

```diff
@@ -1,229 +1,24 @@
-
-
-def _scalar(x):
-    """Convert a scalar tensor into a Python value."""
-    assert x.numel() == 1
-    return x.item()
-
-
-def _if_scalar_type_as(g, self, tensor):
-    """
-    Convert self into the same type of tensor, as necessary.
-
-    We only support implicit casting for scalars, so we never
-    actually need to insert an ONNX cast operator here; just
-    fix up the scalar.
-    """
-    if isinstance(self, torch._C.Value):
-        return self
-    elif tensor.type().kind() == "DimensionedTensorType" or tensor.type().kind() == "CompleteTensorType":
-        ty = tensor.type().scalarType().lower()
-        return getattr(self, ty)()
-    else:
-        return self
-
-
-def _is_value(x):
-    return isinstance(x, torch._C.Value)
-
-
-def _is_tensor_list(x):
-    return x.type().isSubtypeOf(ListType.ofTensors())
-
-
-def _unimplemented(op, msg):
-    warnings.warn("ONNX export failed on " + op + " because " + msg + " not supported")
-
-
-def _try_get_scalar_type(*args):
-    for arg in args:
-        try:
-            return arg.type().scalarType()
-        except RuntimeError:
-            pass
-    return None
-
-
-# ---------------------------------------------------------------------
-# ONNX operator version
-# ---------------------------------------------------------------------
-
-# READ ME BEFORE EDITING _default_onnx_opset_version:
-#
-# The variable below controls which ONNX operator set version we are
-# targeting. THIS VARIABLE HAS SEMANTIC EFFECT! Say a breaking
-# change occurred in version 8. As long as this variable < 8, you can
-# export models targeting the old behavior. However, if you bump
-# this variable to 8 or later, the breaking change will take into effect:
-# you MUST adjust any symbolic affected by breaking changes. The ONNX
-# spec publishes a *comprehensive* list of BC-breaking changes for every
-# operator revision at:
-#
-#   https://github.com/onnx/onnx/blob/master/docs/Changelog.md
-#
-# Please be sure to go through and check all of our implementations here before
-# increasing this number. This includes symbolic definitions NOT in this
-# file, so grep for "OpName" (with quotes)
-#
-# Besides, opset_version can be specified in the invocation of export()
-# and export_to_pretty_string(), and _export_onnx_opset_version will be set
-# and the symbolic functions should check it to determine the behavior
-# of the exporter.
-
-
-_default_onnx_opset_version = 9
-_onnx_master_opset = 10
-_onnx_stable_opsets = [9]
-_export_onnx_opset_version = _default_onnx_opset_version
-
-
-def _set_opset_version(opset_version):
-    global _export_onnx_opset_version
-    if opset_version == _export_onnx_opset_version:
-        return
-    if opset_version in _onnx_stable_opsets + [_onnx_master_opset]:
-        _export_onnx_opset_version = opset_version
-        return
-    raise ValueError("Unsupported ONNX opset version: " + str(opset_version))
 
+# EDITING THIS FILE? READ THIS FIRST!
+# see Note [Edit Symbolic Files]
```

Comment:
tell user where to find the notes?

Reply:
Add some comment for the opset detaisl, such as when the opset is supported by which onnx release version, and release date and so on.